src/components/Messenger/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import { useChat } from '../../'
+import STATES from '../../utils/states'
+
+const styles = {
+  button: {
+    appearance: 'none',
+    background: 'none',
+    borderRadius: '50%',
+    bottom: '18pt',
+    display: 'inline',
+    height: '45pt',
+    padding: '0px',
+    position: 'fixed',
+    right: '18pt',
+    top: 'auto',
+    width: '45pt',
+    zIndex: '2147483646',
+    overflow: 'hidden',
+    boxShadow: '0 3px 12px rgba(0, 0, 0, .15)',
+    transition: 'box-shadow 150ms linear',
+    cursor: 'pointer',
+    outline: 'none',
+    userSelect: 'none'
+  }
+};
+
+const Provider = () => {
```

Comment:
It looks like we need to provide customisation options here, according to [the docs](https://developers.facebook.com/docs/messenger-platform/discovery/customer-chat-plugin#customization) you can customise the color and some greetings (although I'm not sure how these work in the preload phase and if we need to include them when loading the script)